fastn-xtask/src/run_template.rs

```diff
@@ -0,0 +1,86 @@
+use std::env;
```

Comment:
There is a lot of duplication in these run_* methods, they all do the same but operate on a different folder. So create some run helper and call it from the three run_* functions.